src/pip/_internal/operations/prepare.py

```diff
@@ -515,55 +515,51 @@ def prepare_linked_requirements_more(self, reqs, parallel_builds=False):
 
```

Comment:
If we have some `reqs` to download, we might want to have a log line explaining that we are finishing the downloads here with an other `indent_log()` ?

This would transform:
```
$ pip --use-feature=2020-resolver --use-feature=fast-deps  download scipy
WARNING: pip is using lazily downloaded wheels using HTTP range requests to obtain dependency information. This experimental feature is enabled through --use-feature=fast-deps and it is not ready for production.
Collecting scipy
  Obtaining dependency information from scipy 1.5.2
Collecting numpy>=1.14.5
  Obtaining dependency information from numpy 1.19.2
Downloading scipy-1.5.2-cp38-cp38-manylinux1_x86_64.whl (25.7 MB)
   |████████████████████████████████| 25.7 MB 8.5 MB/s 
Downloading numpy-1.19.2-cp38-cp38-manylinux2010_x86_64.whl (14.5 MB)
   |████████████████████████████████| 14.5 MB 25.3 MB/s 
Saved ./scipy-1.5.2-cp38-cp38-manylinux1_x86_64.whl
Saved ./numpy-1.19.2-cp38-cp38-manylinux2010_x86_64.whl
Successfully downloaded scipy numpy
```
to
```
$ pip --use-feature=2020-resolver --use-feature=fast-deps  download scipy
WARNING: pip is using lazily downloaded wheels using HTTP range requests to obtain dependency information. This experimental feature is enabled through --use-feature=fast-deps and it is not ready for production.
Collecting scipy
  Obtaining dependency information from scipy 1.5.2
Collecting numpy>=1.14.5
  Obtaining dependency information from numpy 1.19.2
Completing dependencies download
  Downloading scipy-1.5.2-cp38-cp38-manylinux1_x86_64.whl (25.7 MB)
     |████████████████████████████████| 25.7 MB 8.5 MB/s 
  Downloading numpy-1.19.2-cp38-cp38-manylinux2010_x86_64.whl (14.5 MB)
     |████████████████████████████████| 14.5 MB 25.3 MB/s 
Saved ./scipy-1.5.2-cp38-cp38-manylinux1_x86_64.whl
Saved ./numpy-1.19.2-cp38-cp38-manylinux2010_x86_64.whl
Successfully downloaded scipy numpy
```
?

Reply:
Thanks for the review!  Would it be OK to delay to GH-8771?  I have no problem implemented what you're suggesting though (nor I have any strong opinion favoring either the this PR's current implementation or your suggestion).

Reply:
> Would it be OK to delay to GH-8771?

Sure :)